README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Description
 
-Warden is an authentication middleware service that runs on each node in the vault
+Warden is an authentication middleware service that runs on each node in the Vault
 cluster. It uses a privileged Vault token to provision orphaned tokens for [tokend][].
 the orphaned token has a specific role that grats it access to other services.
 
```

Comment:
Maybe it'd be less clunky to open with "Warden, a Node.js service, is an authentication.....".

This line below seems sad and lonely.